src/transport/AdminPairingTable.h

```diff
@@ -190,8 +228,17 @@ class DLL_EXPORT AdminPairingTable
 
     AdminPairingInfo * FindAdmin(AdminId adminId);
 
+    AdminPairingInfo * FindAdmin(FabricId fabricId);
```

Comment:
These methods seem to assume AdminId is a different type than FabricId. I wonder if we get lukcy by having different int64 and int32 types here.

I am concerned that 'FindAdmin(1)' is very much ambigous. Can we use differentNaming to distinguish and make it clear> Like `FindAdminWithId` and `FindAdminForNode` ?

Why do we have 3 different admin finding methods by fabric/node/vendor? Could we have just one with fallbacks? Or use kUndefined constants for specifying 'any'?

Reply:
done